contributor-docs/testing.md

```diff
@@ -74,6 +74,21 @@ To make assertions about the elements we use [Jest](https://jestjs.io/) and [jes
 
 \*: Read about the [differences between `fireEvent` and `UserEvent`](https://testing-library.com/docs/user-event/intro/#differences-from-fireevent).
 
+### Snapshots
+
+We are slowly moving away from using `Jest` snapshots as a way to test visual changes of our components. You can read more about the decision in this [ADR](https://github.com/primer/react/blob/main/contributor-docs/adrs/adr-011-snapshot-tests.md). We are in the proces of migrating our existing snapshot tests to use [Playwright](https://playwright.dev/) for visual regression testing. If you are writing a new test and you need to test the visual changes of the component, please refer to the [Visual Regression Tests](#visual-regression-tests) section.
+
+#### Updating `theme-preval` snapshots
+
+If you need to update the `theme-preval` snapshots, you will need to run the following commands:
+
+```sh
+npm run build
+npm run test -- -u
+```
+
+After running that command, make sure to update the cache-busting timestamp in [src/theme-preval.js](https://github.com/primer/react/blob/main/src/theme-preval.js)
```

Comment:
Might be nice to add an example of how to update the timestamp here

Reply:
Oh I thought it was just updating [this line](https://github.com/primer/react/blob/main/src/theme-preval.js#L3) manually 🙈 (Because it is such an accurate time on the file now.)

Is it done with `Date.now()` or something?

Reply:
I just do it manually too. I was think we could add a diff block just to be more clear. Something like this:

```diff
- // Cache bust: 2022-02-23 12:00:00 GMT (This file is cached by our deployment tooling, update this timestamp to rebuild this file)
+ // Cache bust: 2023-02-24 12:00:00 GMT (This file is cached by our deployment tooling, update this timestamp to rebuild this file)
```

Reply:
Oh gotcha! I'll add that 😊

Reply:
Done!